# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1539889685 14400
#      Thu Oct 18 15:08:05 2018 -0400
# Node ID 7fa5ee67094aadd71e4376c8afb82d4ae892f29a
# Parent  ac0b3c09c3dbb50b66af7dafbbfe56889fb8597c
* fminsearch.m: Suppress output in test.

diff --git a/scripts/optimization/fminsearch.m b/scripts/optimization/fminsearch.m
--- a/scripts/optimization/fminsearch.m
+++ b/scripts/optimization/fminsearch.m
@@ -520,20 +520,20 @@ endfunction
 %! assert (x, 4.8750, 1e-4);
 %! x = fminsearch (@sin, 3, optimset ("MaxFunEvals", 18, "Display", "none"));
 %! assert (x, 4.7109, 1e-4);
 
 %!test
 %! problem.objective = @sin;
 %! problem.x0 = 3;
 %! problem.solver = "fminsearch";
-%! problem.options = optimset ("MaxIter", 3, "Display", "none")
+%! problem.options = optimset ("MaxIter", 3, "Display", "none");
 %! x = fminsearch (problem);
 %! assert (x, 4.8750, 1e-4);
-%! problem.options = optimset ("MaxFunEvals", 18, "Display", "none")
+%! problem.options = optimset ("MaxFunEvals", 18, "Display", "none");
 %! x = fminsearch (problem);
 %! assert (x, 4.7109, 1e-4);
 
 %!test
 %! c = 1.5;
 %! assert (fminsearch (@(x) x(1).^2 + c*x(2).^2, [1;1]), [0;0], 1e-4);
 
 ## additional input argument
